components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/listener/OrganizationManagerListener.java

```diff
@@ -39,9 +40,15 @@ void postGetOrganization(String organizationId, Organization organization)
             throws OrganizationManagementException;
 
     void preDeleteOrganization(String organizationId) throws OrganizationManagementException;
-    
+
+    @Deprecated
     void postDeleteOrganization(String organizationId) throws OrganizationManagementException;
 
+    default void postDeleteOrganization(String organizationId, int organizationDepthInHierarchy)
+            throws OrganizationManagementException{
+        throw new NotImplementedException("This method is not implemented.");
```

Comment:
leave new line.
NotImplementedException() should be one defined by us. 
`org.apache.commons.lang.NotImplementedException` is used by `org.apache.commons.lang` project to throw errors for functionality that they haven't implemented

Reply:
eg: https://github.com/wso2/carbon-identity-framework/blob/master/components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/exception/NotImplementedException.java

Reply:
Addressed with commit